scripts/atomizer-loop.py

```
@@ -0,0 +1,113 @@
+                        cwd=working_dir,
+                        env=env,
+                        stdout=subprocess.PIPE,
+                        stderr=subprocess.PIPE
+                    )
+
+                    stdout, stderr = process.communicate()
+
+                    print("STDOUT:", stdout.decode())
+                    print("STDERR:", stderr.decode())
+                    
+                    cursor = connection.cursor()
```

Ruff (0.11.9)

Local variable `cursor` is assigned to but never used

Remove assignment to unused variable `cursor`

(F841)